### PR TITLE
{lang,tools}[GCCcore/10.2.0] onedrive v2.4.11, LDC v1.25.1

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

depends on ~#12692~

### CODE REVIEW

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node381.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/66a6ec71b19208155ee97e67011a1920 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in total)
nuc.lan - Linux Fedora 33, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.9.2
See https://gist.github.com/94e760b6e20e506cb554ee19a292bce8 for a full test report.

**Comment by verdurin** [comment]
@smoors dependency problems?:
```
checking for x86_64-pc-linux-gnu-pkg-config... no
checking for pkg-config... /usr/bin/pkg-config
checking pkg-config is at least version 0.9.0... rpm: /mnt/ds918plus/eb/nuc-fedora32/software/XZ/5.2.5-GCCcore-10.2.0/lib/liblzma.so.5: version `XZ_5.2' not found (required by /lib64/librpmio.so.9)
/usr/bin/pkg-config: line 8: /usr/bin/-pkg-config: No such file or directory
no
checking for dmd... /mnt/ds918plus/eb/nuc-fedora32/software/LDC/1.25.1-GCCcore-10.2.0-linux-x86_64/bin/ldc2
checking version of D compiler... 1.25.1
checking for curl... no
configure: error: in `/mnt/ds918plus/eb/nuc-fedora32/build/onedrive/2.4.11/GCCcore-10.2.0/onedrive-2.4.11':
configure: error: The pkg-config script could not be found or is too old.  Make sure it
is in your PATH or set the PKG_CONFIG environment variable to the full
path to pkg-config.

Alternatively, you may set the environment variables curl_CFLAGS
and curl_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.

To get pkg-config, see <http://pkg-config.freedesktop.org/>.
See `config.log' for more details
 (at easybuild/tools/run.py:537 in parse_cmd_output)
 ```

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
easybuild-c7.novalocal - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/575286b3adf85f25140e3f286a50223f for a full test report.

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 20 out of 20 (2 easyconfigs in total)
centos7.lan - Linux centos linux 7.9.2009, x86_64, Intel Core Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/46cb77a39b58725a589646067e58e218 for a full test report.

File: easybuild/easyconfigs/l/LDC/LDC-1.25.1-GCCcore-10.2.0-linux-x86_64.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+easyblock = 'Tarball'
+
+name = 'LDC'
+version = '1.25.1'
+versionsuffix = '-linux-x86_64'
```
Comment (boegel) [inline easybuild/easyconfigs/l/LDC/LDC-1.25.1-GCCcore-10.2.0-linux-x86_64.eb:5]: #12692 is a from source build, I consider that a better option?

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 3 out of 3 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/dc53780884738d002c5d51bab25b7c3e for a full test report.

File: easybuild/easyconfigs/o/onedrive/onedrive-2.4.11-GCCcore-10.2.0.eb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    ('binutils', '2.35'),
+    ('LDC', '1.25.1'),
+]
+
+dependencies = [
+    ('cURL', '7.72.0'),
+    ('SQLite', '3.33.0'),
+]
+
+_compldir = '%(installdir)s/share/tab_completion'
```
Comment (verdurin) [inline easybuild/easyconfigs/o/onedrive/onedrive-2.4.11-GCCcore-10.2.0.eb:27]: @smoors oughtn't this to have the `local` prefix?

**Review by verdurin — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/o/onedrive/onedrive-2.4.11-GCCcore-10.2.0.eb (hunk 2 above)
Comment (smoors) [reply easybuild/easyconfigs/o/onedrive/onedrive-2.4.11-GCCcore-10.2.0.eb:27]: @verdurin both `_` and `local_` prefixes are allowed, see https://docs.easybuild.io/en/latest/Easyconfig-files-local-variables.html#recommended-naming-scheme-for-local-variables-in-easyconfig-files

Comment (verdurin) [reply easybuild/easyconfigs/o/onedrive/onedrive-2.4.11-GCCcore-10.2.0.eb:27]: Thought you were unlikely to have got this wrong, but better to ask...

**Review by verdurin — APPROVED** [review]

**Comment by verdurin** [comment]
Going in, thanks @smoors!

File: easybuild/easyconfigs/o/onedrive/onedrive-2.4.11-GCCcore-10.2.0.eb (hunk 2 above)
Comment (smoors) [reply easybuild/easyconfigs/o/onedrive/onedrive-2.4.11-GCCcore-10.2.0.eb:27]: yeah, the tests would probably fail it it was wrong.

**Pull request merged**